src/Geo/Bounds.ts

```diff
@@ -84,3 +99,51 @@ export function createBoundsFromLocationAndSize(
 		east: ne.lng
 	} as Bounds;
 }
+
+export function getBoundsCenterLocation(bounds: Bounds): Location {
+	const lat: number =  (bounds.north + bounds.south) / 2;
+	let lng: number = (bounds.west + bounds.east) / 2;
+
+	if (bounds.west > bounds.east) {
+		lng = lng > 0 ? lng - 180 : 180 - lng;
+		lng = lng <= 180 ? lng : 360 - lng;
```

Comment:
I think both lines could be simplified to `lng = lng > 0 ? lng - 180 : 180 + lng;`